src/Trace/Link.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Trace;
+
+use Google\Cloud\Core\ArrayTrait;
+
+/**
+ * This plain PHP class represents a Link resource.
```

Comment:
Links out to the documentation for the various resources represented would be quite useful. For instance, inside this docblock:

```
* @see https://cloud.google.com/trace/docs/reference/v2/rest/v2/Links Links
```